src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -60,16 +61,22 @@
 
 .pf-c-dropdown__toggle {
   position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
   min-width: var(--pf-c-dropdown__toggle--MinWidth);
   min-height: var(--pf-c-dropdown__toggle--MinHeight);
   padding: var(--pf-c-dropdown__toggle--PaddingTop) var(--pf-c-dropdown__toggle--PaddingRight) var(--pf-c-dropdown__toggle--PaddingBottom) var(--pf-c-dropdown__toggle--PaddingLeft);
   font-size: var(--pf-c-dropdown__toggle--FontSize);
   font-weight: var(--pf-c-dropdown__toggle--FontWeight);
   line-height: var(--pf-c-dropdown__toggle--LineHeight);
-  color: var(--pf-c-dropdown__toggle--Color);
   background-color: var(--pf-c-dropdown__toggle--BackgroundColor);
   border: none;
 
+  & > * {
```

Comment:
I would expect `color` on the parent to cascade into it's children, but looks like it's because `color` on  `[class^=pf-]` is overriding inheriting color from the parent. I'm wondering if we can write this a different way so that doesn't happen. Do you think it's worth opening an issue to investigate?

Reply:
ya I had the same thought. We could investigate further ya, might be a good general system improvement.

Reply:
https://github.com/patternfly/patternfly-next/issues/976